system-tests/tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/utils/FileTransferSimulationUtils.java

```diff
@@ -77,23 +85,24 @@ public static ChainBuilder contractNegotiationAndFileTransfer(String providerUrl
      * Saves the Contract Negotiation Request ID into the {@see CONTRACT_NEGOTIATION_REQUEST_ID} session key.
      *
      * @param providerUrl       URL for the Provider API, as accessed from the Consumer runtime.
-     * @param contractAgreement Contract agreement definition in JSON format.
      */
-    private static ChainBuilder startContractAgreement(String providerUrl, String contractAgreement) {
-        String connectorAddress = format("%s/api/v1/ids/data", providerUrl);
+    private static ChainBuilder startContractAgreement(String providerUrl, String apiKey) {
```

Comment:
The second argument is not needed regarding the lag of the `auth-tokenbased` module in the dependencies of the control plane